cpp/src/plasma/lib/java/org_apache_arrow_plasma_PlasmaClientJNI.cc

```diff
@@ -241,3 +241,22 @@ JNIEXPORT jlong JNICALL Java_org_apache_arrow_plasma_PlasmaClientJNI_evict(
 
   return static_cast<jlong>(evicted_bytes);
 }
+
+JNIEXPORT jobjectArray JNICALL
+Java_org_apache_arrow_plasma_PlasmaClientJNI_list(JNIEnv* env, jclass cls, jlong conn) {
+  plasma::PlasmaClient* client = reinterpret_cast<plasma::PlasmaClient*>(conn);
+  plasma::ObjectTable objectTable;
+  client->List(&objectTable);
```

Comment:
don't ignore status here, see other code for examples of throwing an exception.

Reply:
modified.